### PR TITLE
New posts page fixed

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

**Comment by jimrandomh** [comment]
Looking at (eg) [The Rocket Alignment Problem](https://www.lesswrong.com/posts/Gg9a4y8reWKtLe3Tn/the-rocket-alignment-problem), the "Reign of Terror" moderation guidelines were replaced with regular frontpage guidelines, but they shouldn't be.

**Comment by jimrandomh** [comment]
If you are small/mobile width there are no items in the "..." menu, the "..." is still shown, and pops up a zero-item menu.

**Comment by jimrandomh** [comment]
Not a new issue, but now that I'm looking closely: the post score and left-side metadata are a shade of grey that's on the wrong side of the legibility guidelines. (`rgba(0,0,0, 0.5)`).

File: packages/lesswrong/components/comments/ModerationGuidelinesBox.jsx [hunk 1]
```diff
@@ -1,55 +1,125 @@
...
 import classNames from 'classnames'
-import Users from 'meteor/vulcan:users';
+
+const styles = theme => ({
+  root: {
+    marginBottom: 10,
+    padding: '8px 14px 1px 14px',
+    cursor: 'pointer',
+    '&:hover': {
+      backgroundColor: 'rgba(0,0,0,0.02)'
```
Comment (jimrandomh) [inline packages/lesswrong/components/comments/ModerationGuidelinesBox.jsx]: I don't like this hover effect. It's right on the border between perceptible and imperceptible, which makes it a bit unsettling when it changes.

Comment (jimrandomh) [reply packages/lesswrong/components/comments/ModerationGuidelinesBox.jsx]: (This varies based on which monitor I view it on; the three monitors I have in front of me vary quite a bit in how much difference there is between 1.0 and .98 in their tone map.)

**Comment by jimrandomh** [comment]
Note, this one depends on a matching change to Vulcan (because of a weird Vulcan bug involving `Utils.dashToCamel` which is suddenly exposed).

File: packages/lesswrong/components/comments/CommentsItem/BanUserFromAllPostsMenuItem.jsx [hunk 2]
```diff
@@ -29,7 +24,7 @@ class BanUserFromAllPostsMenuItem extends PureComponent {
 
   render() {
     const { currentUser, post} = this.props
-    if (Users.canModeratePost(currentUser, post) && post.user && Users.canModeratePost(post.user, post)) {
+    if (Users.canModeratePost(currentUser, post) && post.frontpageDate && Users.owns(currentUser, post)) {
```
[the remark this answers is not in the document]
Comment (Discordius) [reply packages/lesswrong/components/comments/CommentsItem/BanUserFromAllPostsMenuItem.jsx]: Alas, we don't have access to that data here, so I will leave it as is for now. Maybe we want to have a Dialog some other time that independently loads the data.

**Comment by Discordius** [comment]
Also, dependent on: https://github.com/LessWrong2/Vulcan/pull/41

**Pull request merged**